LICENSE

```diff
@@ -27,7 +27,9 @@ The Chapel implementation is composed of two categories of code:
      vim              vim-based syntax coloring                    VIM
 
    third-party/
-     chpldoc-venv    python packages supporting chpldoc
```

Comment:
Note that there is a copy of the license in chapel-doc for chapel.cray.com. I'm not sure if we update that license automatically when we release or if it needs to be updated now. Maybe @bradcray knows.